eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/utils/charsequence/Html.java

```diff
@@ -15,31 +15,30 @@
  * limitations under the License.
  */
 
-package com.schedjoules.eventdiscovery.framework.utils.strings;
+package com.schedjoules.eventdiscovery.framework.utils.charsequence;
 
-import android.content.Context;
-import android.support.annotation.StringRes;
+import android.widget.TextView;
+
+import com.schedjoules.eventdiscovery.framework.utils.factory.Factory;
 
 
 /**
- * Basic implementation for {@link Strings}.
+ * Represents a Html 'formatted' text to be displayed with a {@link TextView}.
  *
  * @author Gabor Keszthelyi
  */
-public final class BasicStrings implements Strings
+public final class Html extends AbstractCachingCharSequence
 {
-    private final Context mAppContext;
-
-
-    public BasicStrings(Context context)
+    public Html(final String inputSource)
     {
-        mAppContext = context.getApplicationContext();
+        super(new Factory<CharSequence>()
+        {
+            @Override
+            public CharSequence create()
+            {
+                return new TrailingLinesTrimmed(android.text.Html.fromHtml(inputSource));
```

Comment:
This doesn't work as expected. It means all styles are dropped, because TrailingLinesTrimmed is not a Spannable anymore. `Html` has to implement the `Spannable` interface to make this work. To trim a Spannable, check this out: https://gist.github.com/mrmike/77fe2a6a357e78108115